From f4ab3140f43b7da0e8d2627f02698ea90a9788eb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ville=20Syrj=C3=A4l=C3=A4?= <ville.syrjala@linux.intel.com>
Date: Tue, 24 Sep 2013 21:26:18 +0300
Subject: drm/i915: Eliminate one indent leel from vlv_find_best_dpll
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Use 'continue' to get rid of one indent level in vlv_find_best_dpll()

Signed-off-by: Ville Syrjälä <ville.syrjala@linux.intel.com>
Reviewed-by: Mika Kuoppala <mika.kuoppala@intel.com>
Signed-off-by: Daniel Vetter <daniel.vetter@ffwll.ch>
(cherry picked from commit 43b0ac531745c3b7cb523dbd2b3f80a2a97ca675)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/gpu/drm/i915/intel_display.c | 40 +++++++++++++++++++-----------------
 1 file changed, 21 insertions(+), 19 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_display.c b/drivers/gpu/drm/i915/intel_display.c
index 6508419fb32a..65318dbce96b 100644
--- a/drivers/gpu/drm/i915/intel_display.c
+++ b/drivers/gpu/drm/i915/intel_display.c
@@ -699,25 +699,27 @@ vlv_find_best_dpll(const intel_limit_t *limit, struct drm_crtc *crtc,
 					       refclk) / (2*refclk));
 					m = m1 * m2;
 					vco = updrate * m;
-					if (vco >= limit->vco.min && vco < limit->vco.max) {
-						ppm = 1000000 * ((vco / p) - fastclk) / fastclk;
-						absppm = (ppm > 0) ? ppm : (-ppm);
-						if (absppm < 100 && ((p1 * p2) > (bestp1 * bestp2))) {
-							bestppm = 0;
-							flag = 1;
-						}
-						if (absppm < bestppm - 10) {
-							bestppm = absppm;
-							flag = 1;
-						}
-						if (flag) {
-							bestn = n;
-							bestm1 = m1;
-							bestm2 = m2;
-							bestp1 = p1;
-							bestp2 = p2;
-							flag = 0;
-						}
+
+					if (vco < limit->vco.min || vco >= limit->vco.max)
+						continue;
+
+					ppm = 1000000 * ((vco / p) - fastclk) / fastclk;
+					absppm = (ppm > 0) ? ppm : (-ppm);
+					if (absppm < 100 && ((p1 * p2) > (bestp1 * bestp2))) {
+						bestppm = 0;
+						flag = 1;
+					}
+					if (absppm < bestppm - 10) {
+						bestppm = absppm;
+						flag = 1;
+					}
+					if (flag) {
+						bestn = n;
+						bestm1 = m1;
+						bestm2 = m2;
+						bestp1 = p1;
+						bestp2 = p2;
+						flag = 0;
 					}
 				}
 			}
-- 
1.8.5.rc3

